### PR TITLE
Adding logary

### PR DESCRIPTION
**Pull request by haf**
Linking to logary

### CODE REVIEW

File: _data/projects.yml [hunk 1]
```diff
@@ -1,3 +1,14 @@
+- name: logary
+  desc: |
```
Comment (shiftkey) [inline _data/projects.yml:2]: I'm rusty as hell on my YAML syntax - does this work, or can we just drop it?

Comment (haf) [inline _data/projects.yml:2]: According to the spec it should work. I don't like lines longer than x chars. Your choice :)

Comment (shiftkey) [inline _data/projects.yml:2]: :thumbsup:

**Comment by shiftkey** [comment]
Thanks!

**Pull request merged**